### ISSUE TITLE
Error: nodejs-18.8.0 PermissionError: [Errno 1] Operation not permitted:

### ISSUE THREAD

**Issue by PlumpMath** [LnL7/nix-darwin#506]
https://github.com/NixOS/nixpkgs/issues/188559

The day before yesterday, the nodejs-18 version cannot be built from August 26, 2022.

Interestingly, it was resolved yesterday on another Intel Mac.
not just on the other side. Why are you doing this? oh my, it's a headache

Thanks.

**Comment by lockejan** [comment]
Are you following unstable nixpkgs channel?
If so, consider switching to a stable channel instead.
Anyway this issue isn't related to nix-darwin but rather nixpkgs.

**Comment by PlumpMath** [comment]
I kept following the master version.
so stoped building like that.

recently switched to the stable version for a while (of course, there are a lot of things that can't be done there... tensorflow-bin etc)
and my computers... There are some systems that rebuild well even in the master version. (Currently, it is still not built on only one laptop ploblem.)

Anyway, if would be use stable, nixpkgs it is certain that it will be completely changed later in the stable version. 
so Without giving up on the nixpkgs-master version, I decided to follow the quick change.
